test/end-to-end/Exception Handling/exception handling.ts

```diff
@@ -0,0 +1,11 @@
+-
+try {
+    // ...
+}
+catch (exceptionType foo) {
```

Comment:
This is invalid TypeScript. You can always validate using http://www.typescriptlang.org/play/index.html.